packages/fuchsia_ctl/lib/src/package_server.dart

```diff
@@ -6,26 +6,38 @@ import 'dart:async';
 import 'dart:convert';
 import 'dart:io';
 
+import 'package:file/file.dart';
+import 'package:file/local.dart';
 import 'package:fuchsia_ctl/fuchsia_ctl.dart';
 import 'package:process/process.dart';
+import 'package:path/path.dart';
```

Comment:
Nit: import it as path, so you end up with path.join

Reply:
Done.